test/e2e/nodenetworkpolicy_test.go

```diff
@@ -891,3 +893,36 @@ func testNodeACNPNestedIPBlockClusterGroupCreateAndUpdate(t *testing.T) {
 	}
 	executeTests(t, testCase)
 }
+
+func testNodeACNPAuditLogging(t *testing.T, data *TestData) {
+	logLabel := "test-labels-thisPartShouldBeTruncated"
+	builder := &ClusterNetworkPolicySpecBuilder{}
+	builder = builder.SetName("acnp-drop-x-to-y-egress").
+		SetPriority(1.0).
+		SetAppliedToGroup([]ACNPAppliedToSpec{{NodeSelector: map[string]string{labelNodeHostname: nodes["x"]}}})
+	builder.AddEgress(ProtocolTCP, &p80, nil, nil, nil, nil, nil, nil, nil, nil, map[string]string{labelNodeHostname: nodes["y"]}, nil,
+		nil, nil, nil, nil, nil, crdv1beta1.RuleActionDrop, "", "", nil)
+	builder.AddEgressLogging(logLabel)
+
+	acnp, err := k8sUtils.CreateOrUpdateACNP(builder.Get())
+	failOnError(err, t)
+	failOnError(data.waitForACNPRealized(t, acnp.Name, policyRealizedTimeout), t)
+
+	expectedLogPrefix := "Antrea:O:Drop:test-labels-:"
+	podXa, err := k8sUtils.GetPodByLabel(getNS("x"), "a")
+	require.NoError(t, err)
+	podXaName := podXa.GetName()
+
+	// Before generating some traffic, there should be no corresponding log on hostNetwork Pod x/a.
+	stdout, _, err := data.RunCommandFromPod(getNS("x"), podXaName, "c80", []string{"dmesg"})
+	require.NoError(t, err)
+	require.NotContains(t, stdout, expectedLogPrefix)
```

Comment:
I think this would fail if I wanted to run the test locally multiple times on the same Kind cluster (e.g., because I am troubleshooting something related to Node NPs).

It should be possible to run the test multiple times on the same cluster. One acceptable approach could be to have a random label, and include a comment in the code explaining why the label is random. There may be better solutions though.

Reply:
I have added a random label. 

Besides, this test cannot be run on github Kind cluster. In a Kind cluster, each Kubernetes Node runs as a Docker container. The iptables LOG target doesn't generate kernel logs inside containers unless /proc/sys/net/netfilter/nf_log_all_netns is set to 1.  However, this setting cannot be modified in GitHub Actions.

Reply:
It's fine to skip that specific test on Kind